### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by im4aLL** [ad-on-is/adonis-autoswagger#84]
**Using** 

- AdonisJS: 6
- adonis-autoswagger: 3.19.0

I am not sure if I am doing something wrong but I just updated it to 3.19.0 today and after the update, I am getting 

```
Maximum call stack size exceeded
    request_id: "yiyidj2y8mjvdvf8pvjg371c"
    x-request-id: "yiyidj2y8mjvdvf8pvjg371c"
    err: {
      "type": "RangeError",
      "message": "Maximum call stack size exceeded",
      "stack":
          RangeError: Maximum call stack size exceeded
              at String.split (<anonymous>)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:99:38)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
              at ExampleGenerator.getSchemaExampleBasedOnAnnotation (/Users/hadi/repos/backend/node_modules/adonis-autoswagger/dist/example.js:168:37)
      "status": 500
    }
```

Any help is appreciated!

**Comment by ad-on-is** [comment]
can you try to pinpoint what causes it on your end?

**Comment by ad-on-is** [comment]
Fixed in 3.21.0
